PaddleAudio/examples/dcase2021_task1b/README.md

```diff
@@ -0,0 +1,111 @@
+- Pedestrian street - street_pedestrian
+- Public square - public_square
+- Street with medium level of traffic - street_traffic
+- Travelling by a tram - tram
+- Travelling by a bus - bus
+- Travelling by an underground metro - metro
+- Urban park - park
+
+### Audio-only
+
+For audio-only scene classification, we extract the same feature as described in  [audioset training example](../audioset_training) and use the same network, namely resent50, except that we add extra fc-layers and one final classification head for the 10-way classification. We also add dropout to reduce over-fitting. See [model.py](model.py) for details.
+
+Training audio-only network: 
+``` sh 
+python train.py --device=0 --task_type=audio_only --config='./assets/config.yaml'
+```
+
+To evaluate using our pre-trained weights, try 
+``` sh 
+python evaluate.py --device=0 --task_type=audio_only --config='./assets/config.yaml'
+```
+
+
+
+### Video-only
+For video-only task, we start with from model 
+#### Convert video to images
+We first extract images at 1 frame per second from the provided video files for further process. Using ffmpeg, this can be done as follows, 
+
+``` sh
+mkdir -p images
+for f in `ls video/*.mp4`; do 
+    file=`echo $f |cut -d . -f 1|cut -d / -f 2`
+    ffmpeg -i $f -vf fps=1 ./images/$file-%02d.png
+done
+```
+The above script assumes that you put all video files under folder './video'.
+
+#### Zero-shot classification
+After converting video to images, we use OpenAI clip to extract class probability in zero-shot manner. We use the 5th frame to represent the whole video. The usage of zero-shot image classification is explained in [OpenAI's implementation](https://github.com/openai/CLIP) in Pytorch, and also in a third-party  [implementation](https://github.com/ranchlai/CLIP.paddle). 
+
+The overall video-only accuracy for zero-shot is 0.772 without any heavy prompt engineering, and the detail results can be found in [assets/clip_zero_shot_results.pkl](./assets/clip_results.pkl)
+
+#### linear-probe classification
+For linear-probe, see [README_LP.md](README_LP.md)
+
+### Audio-visual task
+We can combine audio and video branch in a late-fusion manner. THe network architecture is as follows, 
+![network](./assets/network_arch.png)
+
+The audio-visual task utilizes embedding output form CLIP image encoder. For the convenience of this demo, the embedding is already pickled as ['.assets/file2feature.pkl](.assets/file2feature.pkl). 
+
+
+To train audio-visual network, run
+``` sh 
+python train.py --device=0 --task_type='audio_visual' --config='./assets/config.yaml'
+```
+
+To evaluate using our pre-trained weights, run 
+``` sh 
+python evaluate.py --device=0 --task_type='audio_visual' --config='./assets/config.yaml'
+```
+
+With audioset\[4\] pre-training, mixup\[3\] and spectrogram augmentation techniques, we achieve <b>92.5%</b> overall classification score.  
+
+### Performance
+
+|task|acc|
+|--|--|
+|video-only(linear probe)|0.880|
```

Comment:
Is it possible to submit your result to DECASE to verify the correctness of your results?

Reply:
Still waiting for DCASE challenge 2021 to release the evaluation dataset. Will submit  when available.